apps/03TicTacToe.js

```diff
@@ -23,24 +32,71 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
-function horizontalWin() {
-  // Your code here
+// tic tac toe function
+
+function ticTacToe(row, column) {
+  //determine if the spot on the board is already taken
+
+  if (board[row][column] != " ") {
+    console.log("This space is taken. Please pick again.");
+
+  //if not...place mark, switch players and check for win
+
+  } else {
+    board[row][column] = playerTurn;
+    checkForWin();
+    playerTurn = (playerTurn === 'X') ? 'O' : 'X';
+  };
 }
 
-function verticalWin() {
-  // Your code here
+// check for win function
+
+function checkForWin() {
+  if (horizontalWin() === true || verticalWin() === true || diagonalWin() === true) {
```

Comment:
You can pass in functions to an `if` statement and it will automatically evaluate the 'truthiness' of whatever the functions return.

ex: `if(returnTrueOrFalse()) { return 'I am true!'..`